earth_enterprise/src/fusion/gepublish/geserveradmin.cpp

```diff
@@ -623,6 +630,10 @@ int main(int argc, char* argv[]) {
         throw khException("Target path is not specified.\n");
       }
 
+      if (enable_enhancedsearch && !enable_poisearch) {
```

Comment:
I'd like to see one other error check here - poi search and enhanced search shouldn't be specified unless publishdb is set.

Reply:
Check has been added!